pkg/gui/pty.go

```diff
@@ -87,3 +94,24 @@ func (gui *Gui) newPtyTask(view *gocui.View, cmd *exec.Cmd, prefix string) error
 
 	return nil
 }
+
+func removeExistingTermEnvVars(env []string) (updatedEnv []string) {
+	for _, envVar := range env {
+		if !isTermEnvVar(envVar) {
+			updatedEnv = append(updatedEnv, envVar)
+		}
+	}
+
+	return
+}
```

Comment:
I'm not a fan of named return values (and if I remember right, @jesseduffield isn't either, but I might be wrong about that). It may just be a matter of personal taste, but I'd have written this as follows:
```go
func removeExistingTermEnvVars(env []string) []string {
	return lo.Filter(env, func(envVar string, _ int) bool {
		return !isTermEnvVar(envVar)
	})
}
```

Reply:
Fixed in [0d4b65f](https://github.com/jesseduffield/lazygit/pull/3420/commits/0d4b65f2b806dc5647817bf0116db54deb6c1f69)